test/test-server/org_handlers.go

```diff
@@ -56,8 +58,17 @@ func handleOrgEnvironment(t *testing.T) http.HandlerFunc {
 				StreamGovernanceConfig: &orgv2.OrgV2StreamGovernanceConfig{Package: orgv2.PtrString("ESSENTIALS")}}
 			err := json.NewDecoder(r.Body).Decode(req)
 			require.NoError(t, err)
-			req.Id = orgv2.PtrString(id)
+			if id == "env-595" && req.StreamGovernanceConfig.GetPackage() == "ESSENTIALS" {
+				w.WriteHeader(http.StatusForbidden)
+				resp := errors.ErrorResponseBody{Errors: []errors.ErrorDetail{
+					{Detail: "SR package downgrade is not a supported operation"}},
```

Comment:
```suggestion
					{Detail: "Schema Registry package downgrade is not a supported operation"}},
```

Reply:
I'm gonna reject this one because it already mimics the exact response from the org service

Reply:
Can we change the backend error to say "Schema Registry"? We try to avoid abbreviations in any customer-facing text.